src/Configuration/Provisioning/SubsidyEditView/SubsidyEditView.jsx

```diff
@@ -35,6 +35,19 @@ const SubsidyEditView = () => {
     });
   };
 
+  useEffect(() => {
+    const handleTabClose = (event) => {
+      event.preventDefault();
+      const copyEvent = event;
+      copyEvent.returnValue = true;
+      return copyEvent.returnValue;
```

Comment:
Do we need to make a copy of the event here?
```suggestion
      const copyEvent = event;
      copyEvent.returnValue = true;
      return copyEvent.returnValue;
```

Reply:
Yup! this was to get around the linter rule `no-param-reassign when setting a property on a DOM object`

Reply:
Do we need to set this?  Documentation suggests that `returnValue` is true by default (and also deprecated).
https://developer.mozilla.org/en-US/docs/Web/API/Event/returnValue